content/20.treatments.md

```diff
@@ -617,37 +617,33 @@ Biologics are produced from components of living organisms or viruses. They incl
+Unlike membrane-bound IL-6Rα, which is only found on hepatocytes and some types of leukocytes, gp130 is found on most cells [@doi:10.1007/s00430-006-0019-9].
+When IL-6 binds to sIL-6Rα, the complex can then bind to a gp130 protein on any cell [@doi:10.1007/s00430-006-0019-9].
+The binding of IL-6 to IL-6Rα is termed classical signaling, while its binding to sIL-6Rα is termed trans-signaling [@doi:10.1016/j.cytogfr.2012.04.001; @doi:10.1042/bj3000281; @doi:10.1007/s00430-006-0019-9].
+These two signalling processes are thought to play different roles in health and illness; for example, trans-signalining may play a role in the proliferation of mucosal T-helper TH2 cells associated with asthma, while an earlier step in this proliferation of process, the CD4+-cells towards the TH2 lineage, may be regulated by classical signaling, and IL-6 is known to play a role in Crohn's Disease via trans-, but not classical, signaling [@doi:10.1007/s00430-006-0019-9].
+Both classical and trans-signaling can occur through three independent pathways: the Janus-activated kinase (JAK)-STAT3 pathway, the Ras/Mitogen-Activated Protein Kinases (MAPK) pathway and the Phosphoinositol-3 Kinase (PI3K)/Akt pathway [@doi:10.3389/fimmu.2016.00604].
+These signalling pathways are involved in a variety of different functions, including cell type differentiation, immunoglobulin synthesis, and cellular survival signaling pathways, respectively [@doi:10.3389/fimmu.2016.00604].
+The ultimate result of the IL-6 cascade is to direct transcriptional activity of various promoters of pro-inflammatory cytokines, such as IL-1, TFN, and even IL-6 itself, through the activity of NF-κB [@doi:10.3389/fimmu.2016.00604].
+IL-6 synthesis is tightly regulated both transcriptionally and post-transcriptionally, and it has been shown that viral proteins can enhance transcription of the IL-6 gene by  strengthening the DNA-binding activity between several transcription factors and IL-6 gene-cis-regulatory elements [@doi:10.2492/inflammregen.33.054].
+Therefore, drugs inhibiting the binding of IL-6 to IL-6Rα or sIL-6Rα are of interest for combatting the hyperactive inflammatory response characteristic of the cytokine storm.
+TCZ is a humanized monoclonal antibody that binds both to the insoluble and soluble receptor of IL-6, providing de facto inhibition of the IL-6 immune cascade.
 
 **Current Evidence.**
-Chinese doctors have started a trial enrolling 188 patients of which 14 with severe lung disease have shown clear signs of improvements, according to their results [@url:https://www.genengnews.com/virology/coronavirus/genentech-launches-phase-iii-trial-of-actemra-as-coronavirus-treatment/].
+Tocilizumab is being investigated either as a primary or supplemental treatment in {{ebm_tocilizumab_ct}} COVID-19 clinical trials curently registered at clinicaltrials.gov. 
```

Comment:
@agitter, this was my guess of how to invoke the stored information from the json file -- but I wanted to run it by you!